src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String[] cookieValues;
+  public Cookie userCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "loginStatus";
+    cookieValues = {"out", "null"};
+  }
+
+  public void loginUser() {
+    String AuthID;
+    userCookie = new Cookie(cookieName, cookieValues);
+    
+    // call auth service
+    UserService userService = UserServiceFactory.getUserService();
+
+    // if the user has logged in
+    if (userService.isUserLoggedIn()) {
+      
+      // set the first cookie value status to "in"
+      userCookie.getValue[0] = "in";
+      
+      // get AuthID
+      AuthID = userService.getCurrentUser().getUserId();
+      
+      // find AuthID in DataStore
+      Query query = new Query("User").addFilter("AuthID", FilterOperator.EQUAL, userID);
```

Comment:
Try to see if you can use your UserController to do this query, instead of putting together a new query here.